assets/js/quizQuestions.js

```diff
@@ -0,0 +1,69 @@
+const questionTemplate = {
+  question: "",
+  answers: ["", "", "", ""],
+  correctAnswer: "",
+};
+
+// feel free to add some questions to the arrays a template is above, you can put more of less than for answers but dont have to many
+
+
+const htmlQuestions = [
```

Comment:
I like how you made it really easy to add questions

Reply:
Yeah i try to make it as simple as possible, thats also why i put in separate file so people can find easy, will be good when some more questions are added and its a proper quiz.